x/marker/keeper/msg_server.go

```diff
@@ -808,3 +818,36 @@ func (k msgServer) UpdateSendDenyList(goCtx context.Context, msg *types.MsgUpdat
 
 	return &types.MsgUpdateSendDenyListResponse{}, nil
 }
+
+// AddNetAssetValues adds net asset values to a marker that is in pending state
```

Comment:
I noticed you mention pending state here, but don't see a check for it.

Reply:
Old design.  Removing... Can be gov proposal or anyone with some sort of access on the marker.